docs/modules/data-structures/pages/distributed-data-structures.adoc

```diff
@@ -211,6 +211,10 @@ Hazelcast is designed to create any distributed data structure whenever it is ac
 Therefore, keep in mind that a
 data structure is recreated when you perform an operation on it even after you have destroyed it.
 
+NOTE: If you plan to use the distributed object later, prefer clearing the object using `clear()` method or equivalent instead of destroying it.
+
+WARNING: Do not use distributed object concurrently with `destroy()` invocation. This may make the object not fully usable.
```

Comment:
```suggestion
WARNING: Do not use the distributed object concurrently with the `destroy()` invocation, as this can result in the object not being fully usable.
```